src/main/java/io/vlingo/schemata/infra/persistence/SchemataObjectStore.java

```diff
@@ -0,0 +1,210 @@
+                        "vlingo_schemata",
+                        MapQueryExpression.using(Unit.class, "find", MapQueryExpression.map("id", "id")),
+                        mappersLookup.get(SchemaState.class));
+        registry.register(schemaInfo);
+
+        final Info<SchemaVersion> schemaVersionInfo =
+                new Info(
+                        objectStore,
+                        SchemaVersionState.class,
+                        "vlingo_schemata",
+                        MapQueryExpression.using(Unit.class, "find", MapQueryExpression.map("id", "id")),
+                        mappersLookup.get(SchemaVersionState.class));
+        registry.register(schemaVersionInfo);
+    }
+
+    private void initializeDatabase(final Configuration configuration) throws Exception {
+        jdbi = JdbiOnHSQLDB.openUsing(configuration);
+        jdbi.createCommonTables();
+
+        this.createOrganizationStateTable();
+        this.createUnitStateTable();
+        this.createContextStateTable();
+        this.createSchemaStateTable();
+        this.createSchemaVersionStateTable();
+    }
+
+    private void createOrganizationStateTable() {
+        jdbi.handle().execute("CREATE TABLE IF NOT EXISTS ORGANIZATION " +
+                "(id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1) PRIMARY KEY, " +
+                "organizationId VARCHAR (50), name VARCHAR(100), description VARCHAR(1024))");
+    }
+
+    private void createUnitStateTable() {
+        jdbi.handle().execute("CREATE TABLE IF NOT EXISTS UNIT " +
+                "(id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1) PRIMARY KEY, " +
+                "unitId VARCHAR(50), organizationId VARCHAR (50), name VARCHAR(100), description VARCHAR(1024))");
+    }
+
+    private void createContextStateTable() {
+        jdbi.handle().execute("CREATE TABLE IF NOT EXISTS CONTEXT " +
+                "(id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1) PRIMARY KEY, " +
+                "contextId VARCHAR(50), unitId VARCHAR(50), organizationId VARCHAR (50), namespace VARCHAR(100), description VARCHAR(1024))");
+    }
+
+    private void createSchemaStateTable() {
+        jdbi.handle().execute("CREATE TABLE IF NOT EXISTS SCHEMA " +
+                "(id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1) PRIMARY KEY, " +
+                "schemaId VARCHAR(50), contextId VARCHAR(50), unitId VARCHAR(50), organizationId VARCHAR (50)" +
+                ", category VARCHAR(25), name VARCHAR(100), description VARCHAR(1024))");
+    }
+
+    private void createSchemaVersionStateTable() {
+        jdbi.handle().execute("CREATE TABLE IF NOT EXISTS SCHEMAVERSION " +
+                "(id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1) PRIMARY KEY, " +
+                "schemaVersionId VARCHAR(50), schemaId VARCHAR(50), contextId VARCHAR(50), unitId VARCHAR(50), organizationId VARCHAR (50)" +
```

Comment:
Also please not that I'm not sure what are really the lengths required for `VARCHAR`